ADO.NET/GettingStarted/SampleApp.csproj

```diff
@@ -16,4 +16,8 @@
     <PackageReference Include="Vertica.Data" Version="24.1.0" />
   </ItemGroup>
 
+  <ItemGroup>

```

Comment:
I've had visualization issues in github in the past, so this might be fine, but it appears to me like the indentation varies widely on these new lines.

Reply:
Your eyes do not deceive you; it's fixed now.